agent/frap.py

```diff
@@ -138,7 +138,7 @@ def relation(self):
         '''
         comp_mask = []
         for i in range(len(self.phase_pairs)):
-            zeros = np.zeros(len(self.phase_pairs) - 1, dtype=np.int)
```

Comment:
Hi @Red-Pheonix, can you make sure this modification is compatible with previous numpy versions as well? An easy fix could be adding a checker on numpy version with if/else - but if you have better solutions, we would really appreciate it!

Reply:
The fix I added should also work in older versions in numpy. Going by the `requirements.txt` file, `np.int64` is also available in `numpy version 1.17`. So no checks are needed.

Reply:
Here is the documentation page listing `np.int64` for `numpy version 1.17`  https://numpy.org/doc/1.17/user/basics.types.html

Reply:
Thanks!